test/functional/interface_rpc.py

```diff
@@ -67,10 +80,23 @@ def test_http_status_codes(self):
         expect_http_status(404, -32601, self.nodes[0].invalidmethod)
         expect_http_status(500, -8, self.nodes[0].getblockhash, 42)
 
+    def test_work_queue_exceeded(self):
+        self.log.info("Test work queue exceeded")
+        self.restart_node(0, ['-rpcworkqueue=1'])
+        got_exceeded_error = []
```

Comment:
This is a list, rather than just a boolean, because a list is a mutable object, and this eliminates the need for the global variable, which seems cleaner. Is it worth a comment explaining that? I tend to think not.

Reply:
nit:
> Is it worth a comment explaining that? I tend to think not.

I don't think so, but it's a bit awkward for the list to be named like a boolean, maybe `exceeded_errors` if you have to retouch ?